### DISCUSSION TITLE
Clarification of 4/8 button remote configuration instructions

### DISCUSSION THREAD

**Discussion by sahafeez** [kuestess/homebridge-platform-insteonlocal#224] [General]
@kuestess I am a little confused on the setup for the 8 button remote (not the wired in wall one, the portable). From the docs I have come up with this:
           
```
               {
                    "name": "4/8 Button Remote",
                    "deviceID": "44.B9.68",
                    "dimmable": "no",
                    "deviceType": "remote"
                },
                {
                    "name": "Remote C - All Living Room Lights",
                    "deviceID": "44.B9.68",
                    "dimmable": "yes",
                    "deviceType": "remote",
                    "groupID": "70",
                    "keypadbtn": "C",
                    "groupMembers": "41.06.DC,41.0C.E9,41.21.6C"
                },
```

Question:

The remote is 8 button in the 2 button on/off mode. So for the example above for keypadbtn that is technically button C and D? 

The group ID I used maps to a scene. 

```
                {
                    "name": "Scene - All Living Room Lights",
                    "deviceID": "31.18.75",
                    "dimmable": "yes",
                    "deviceType": "scene",
                    "groupID": "70",
                    "groupMembers": "41.06.DC,41.0C.E9,41.21.6C"
                },

```

Given the number of new people with the shutdown of Insteon, I can see you are getting a ton of Question. I wonder if it would be worth it for you to setup a google group or the like for everyone to join and maybe help each other vs. opening issues all the time?

**Comment by kuestess** [comment]
@sahafeez Yes, the volume of issues has definitely increased - like your idea of creating a place for community interaction.  That said, I've opened up the 'Discussions' section of the repo and have moved this issue over to be the inaugural discussion :-).  Regarding your question, the `remote` device is really intended for use as a trigger for HomeKit scenes/automations, ie ideally used to trigger something (Insteon or not) when the button state changes.  The easiest way to get what I think you're looking for is to use the second scene config block that you posted above.

**Reply by sahafeez** [reply]
So the remote is programmed to trigger a scene. In this case, #70. What I am trying to sort for is the fact that the status of the scene in HomeKit is not reflected when the remote is used. So I can see all of its members are ON, but the scene icon itself is not marked as on. Is that something that can be achieved?
